Update hashes for `bash` versions

dav3r opened the pull request:
# <!-- Use the title to describe PR changes in the imperative mood --> #

## 🗣 Description ##

This PR updates the known hashes for `bash` versions on our current production systems.

<!-- Describe the "what" of your changes in detail. -->
<!-- To avoid scope creep, limit changes to a single goal. -->

## 💭 Motivation and context ##

As our deployed instances move to newer versions of `bash`, we want to update their hashes in this repository so that we can be assured that our code is indeed functioning as intended.

<!-- Why is this change required? -->
<!-- What problem does this change solve? How did you solve it? -->
<!-- Mention any related issue(s) here using appropriate keywords such -->
<!-- as "closes" or "resolves" to auto-close them on merge. -->

## 🧪 Testing ##

I verified that these updated hashes are indeed being detected when I scanned our Production instances.

<!-- How did you test your changes? How could someone else test this PR? -->
<!-- Include details of your testing environment, and the tests you ran to -->
<!-- see how your change affects other areas of the code, etc. -->

<!--
## 📷 Screenshots (if appropriate) ##

Uncomment this section if a screenshot is needed.

-->

## ✅ Pre-approval checklist ##

<!-- Remove any of the following that do not apply. -->
<!-- Draft PRs should have one or more unchecked boxes. -->
<!-- If you're unsure about any of these, don't hesitate to ask. -->
<!-- We're here to help! -->

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] *All* future TODOs are captured in issues, which are referenced
      in code comments.
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All new and existing tests pass.

## ✅ Post-merge checklist ##

<!-- Remove any of the following that do not apply. -->

- [x] Add a tag.

dav3r committed:
Update hashes for bash versions on current production systems

dav3r committed:
Bump version from 1.4.0 to 1.4.1

jsf9k reviewed (approved):
_Bash hash!_

mcdonnnj commented on src/ioc_scan/ioc_scanner.py:
```diff
@@ -42,8 +42,10 @@
 few things that should hit:
 GNU bash, version 3.2.57(1)-release (x86_64-apple-darwin18)
```
Just as a thought should we include zsh for macOS since it is the new default?

dav3r replied:
Yes, that's an excellent idea - added in https://github.com/cisagov/ioc-scanner/pull/42/commits/8925967c197821cc3fd2278b578dcbdc3b180543.

mcdonnnj reviewed (approved):
Makes sense. I had one thought that you can feel free to decline.

dav3r committed:
Add known hash for zsh

Since zsh is the current Mac default shell, we figured it would be
good to included it here.

Co-authored-by: Nick <50747025+mcdonnnj@users.noreply.github.com>
